### ISSUE TITLE
GUI failure to set configurations on Ubuntu 20.04

### ISSUE THREAD

**Issue by branilson** [g-arjones/clevo-xsm-wmi#7]
GUI doesn' t work on Ubuntu 20.04. I did not test the GUI in other distributions. 

When pressed Apply button, the following messages appear in terminal:
```console
[clevo-xsm-wmi] Failed to set brightness
[clevo-xsm-wmi] Failed to set colour
[clevo-xsm-wmi] Failed to set mode
[clevo-xsm-wmi] Failed to set state
```
Everything is working fine via keyboard hotkeys.

**Comment by g-arjones** [comment]
Did you install the utility with `sudo make install`?

**Comment by branilson** [comment]
Yes, I did. I've followed the README.MD. But, now I see that the GUI works if I run it with sudo.

Sorry, I only saw your comment now.
